operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -138,6 +134,11 @@ func (r *TenantReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		}
 	}
 
+	if tn.Status.Subscriptions == nil {
+		// make initial len is 2 (keycloak and nextcloud)
+		tn.Status.Subscriptions = make(map[string]crownlabsv1alpha2.SubscriptionStatus, 2)
+	}
```

Comment:
@QcFe Moved here because it could be overridden by the Update operation above, leading to a nil map, and thus causing a panic in the tests. Honestly, it is unclear to me how it could work without issues before.

Reply:
😳